eng/scripts/build.ps1

```diff
@@ -9,7 +9,7 @@ param(
     [switch]$format,
     [switch]$tidy,
     [string]$config = "autorest.md",
-    [string]$goExtension = "@autorest/go@4.0.0-preview.35",
+    [string]$goExtension = "@autorest/go@4.0.0-preview.36",
```

Comment:
It might be worth adding some automation that will submit a PR to increment this version whenever a new version is published. We do that in some other languages.

Reply:
I've created an [issue](https://github.com/Azure/azure-sdk-for-go/issues/16879) and will work with it next week.